Annex60/BoundaryConditions/WeatherData/BaseClasses/Examples/GetAbsolutePath.mo

```diff
@@ -1,18 +1,15 @@
 within Annex60.BoundaryConditions.WeatherData.BaseClasses.Examples;
 model GetAbsolutePath "Test model to get the absolute path of a URI"
   extends Modelica.Icons.Example;
-  parameter String f[:] = {"file://package.mo",
-                           "modelica://Annex60/package.mo",
-                           "package.mo"} "Name of a file that exists";
-  parameter String fAbs[:]=
+  parameter String f = "modelica://Annex60/package.mo"
+   "Name of a file that exists";
+  parameter String fAbs=
     Annex60.BoundaryConditions.WeatherData.BaseClasses.getAbsolutePath(uri=f)
     "Absolute path of f";
-  final parameter Integer dummy = size(f, 1)
+  final parameter Integer dummy = 1
     "Dummy variable, used have a result needed for the unit tests";
```

Comment:
@mwetter Should this rather read "Dummy variable used **to** have a result needed for the unit tests?